src/drive/web/modules/public/LightFileViewer.jsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react'
-import Viewer from 'viewer'
+import { Viewer } from 'cozy-ui/transpiled/react'
 import PropTypes from 'prop-types'
 
 import PublicToolbar from './PublicToolbar'
 import styles from 'drive/web/modules/viewer/barviewer.styl'
 
+import createWorker from 'react-pdf/dist/pdf.worker.entry.js'
+import { pdfjs } from 'react-pdf'
+
+pdfjs.GlobalWorkerOptions.workerPort = createWorker()
```

Comment:
Do you think we need to write a not about that in the UI doc? 

Do you think is there any value to mutualize that? Since we used it also in LightFolder? should we create a `<PublicViewer>` wraping `<UiViewer>` ?

Reply:
> Do you think we need to write a not about that in the UI doc?

Probably uncommon but it can't hurt.

>Do you think is there any value to mutualize that? Since we used it also in LightFolder? should we create a <PublicViewer> wraping <UiViewer> ?

Yup.